res/controllers/Traktor Kontrol S4 MK3.hid.xml

```diff
@@ -9,6 +9,615 @@
+                max="250"
+                step="2"
+                default="50"
+                label="Jogwheel sensitivity in loop mode" />
+            <description>
+                Define the sensitivity factor when the jogwheel is used to move the in and out point of a loop using the Loop Mode.
+            </description>
+            <row>
+                <option
+                    variable="loopEncoderMoveFactor"
+                    type="integer"
+                    min="100"
+                    max="2500"
+                    step="20"
+                    default="500"
+                    label="Loop encoder sensitivity in loop mode">
+                    <description>
+                        Define the sensitivity of the loop encoder when moving loop position in the Loop Mode.
+                    </description>
+                </option>
+                <option
+                    variable="loopEncoderShiftMoveFactor"
+                    type="integer"
+                    min="500"
+                    max="12500"
+                    step="20"
+                    default="2500"
+                    label="Loop encoder sensitivity in loop mode (shift)">
+                    <description>
+                        Define the sensitivity of shift+the loop encoder when moving loop position in the Loop Mode.
+                    </description>
+                </option>
+
+            </row>
+
+            <option
+                variable="wheelSpeedSample"
+                type="integer"
+                min="2"
+                max="80"
+                step="2"
+                default="5"
+                label="Speed sample">
+                <description>
+                    Define how many wheel moves are sampled to compute the speed. The more you have, the more the speed is accurate, but the less responsive it gets in Mixxx.
+                </description>
+            </option>
+            <option
+                variable="baseRevolutionsPerMinute"
+                type="enum"
+                label="Turntable speed">
+                <value label="33.3 RPM" default="true">33.33</value>
+                <value label="45 RPM">45</value>
+                <description>Define how fast the jogwheel would rotate with the default play rate. This will impact scratch sensitivity, motor speed and ring LED pace.</description>
+            </option>
+        </group>
+        <group label="Haptic drive feature (BETA)">
+            <option
+                variable="useMotors"
+                type="boolean"
+                default="false"
+                label="Use beta Haptic Drive features (BETA)">
+                <description>
+                    This will defines whether or not Mixxx is allowed to interact with the motors. &lt;br /&gt;&lt;b&gt;WARNING&lt;/b&gt;: while interacting with motors should be harmless for your device, it isn't possible to give a full guaranty that this will not accelerate wearing off.
```

Comment:
```suggestion
                    This will defines whether or not Mixxx is allowed to interact with the motors. &lt;br /&gt;&lt;b&gt;WARNING&lt;/b&gt;: while interacting with motors should be harmless for your device, it could cause excess wear and tear.
```